cumulusci.yml

```diff
@@ -102,19 +105,36 @@ flows:
                 task: deploy_trial_config
                 options:
                     unmanaged: False
+    qa_org:
```

Comment:
That sounds fine. Correct me if I am wrong, qa_org and dev_org seem to be the same now. Do we still want to have this new flow? It might be some pain to sync these two flows moving forward.

Reply:
That's a fair point, @xinnanlisf. We don't have a consistent pattern for the `qa_org` flow; sometimes it extends `dev_org`, sometimes it replaces it. Right now, K-12's unified (`dev_org`, `qa_org`, and `trial_org` are identical), so one approach would be to move:
- `enable_course_connections`
- `deploy_trial_config`
- `execute_install_apex`

into another flow, then append that flow to both `config_dev` and `config_qa`. Let me sleep on it.

Reply:
@jstvz just a quick follow up on this, are you working on some investigations on this? I feel like maybe we can merge it and work on qa_org specific flow/task later on since this PR has been there for a while.

Reply:
Fair point, @xinnanlisf. I'm fine with merging this as-is.